laws/src/main/scala/algebra/laws/LatticeLaws.scala

```diff
@@ -23,6 +23,7 @@ trait LatticeLaws[A] extends Laws {
   def joinSemilattice(implicit A: JoinSemilattice[A]) = new LatticeProperties(
     name = "joinSemilattice",
     parents = Nil,
+    Rules.serializable(A),
```

Comment:
Yes they should! It's troubling that this test was passing. I'll take a closer look.